### ISSUE TITLE
[BUG] Crash on startup with Macaw's Doors 1.1.0

### ISSUE THREAD

**Issue by milek** [Kitteh6660/DramaticDoors#79]
**Important Information**
Minecraft Version: **1.20.4**
Mod Loader System: **Fabric**
DramaticDoors Version: **1.20.4-3.2.0**

**Describe the bug**
Mod is crashing on startup, before even showing splash screen.
Most likely due to recent change in WaterLogging of some doors. Excerpt from changelog:
```
- Removed Waterlogging for Garage Doors, Portcullis, Stable Door, Shoji Door
```
**To Reproduce**
- Install latest (compatible with 1.20.4) Macaw's Doors mod
- Install DramaticDoors mod
- Launch game

**Expected behavior**
Do no crash.

**Logs**
```
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'mcwdoors'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$2(FabricLoaderImpl.java:388)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:386)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:487)
	at net.minecraft.client.main.Main.main(Main.java:223)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.ExceptionInInitializerError
	at net.kikoz.mcwdoors.MacawsDoors.onInitialize(MacawsDoors.java:28)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 6 more
Caused by: java.lang.IllegalArgumentException: Cannot set property class_2746{name=waterlogged, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{minecraft:air}
	at net.minecraft.class_2688.method_11657(class_2688.java:122)
	at net.minecraft.class_2323.handler$zjc000$dramaticdoors$enhanceConstructor(class_2323.java:541)
	at net.minecraft.class_2323.<init>(class_2323.java:70)
	at net.kikoz.mcwdoors.objects.JapaneseDoors.<init>(JapaneseDoors.java:75)
	at net.kikoz.mcwdoors.init.BlockInit.<clinit>(BlockInit.java:23)
	... 8 more
```

**Comment by Kitteh6660** [comment]
The reason it happened was because back then, Macaw's Doors wasn't yet updated for 1.20.4.

When I get around to working on the mod, I will look into this issue and get it fixed!

**Comment by Kitteh6660** [comment]
The fix is now live in mod version 3.2.1 for Minecraft 1.20.4.
